<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>work</class>
 <widget class="QDialog" name="work">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
    border:2px solid gray;
    border-radius: 10px;
}
QPushButton:hover{
    color:white;
    background: rgb(52, 203, 147);
}  

QPushButton:checked
{
    color:white;
    background:black;
}
</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QcalendarWidget::item:selected {
     border: 1px solid #6a6ea9;
}
QcalendarWidget::item:selected:!active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #ABAFE5, stop: 1 #8588B2);
}
QcalendarWidget::item:selected:active {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #6a6ea9, stop: 1 #888dd9);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_work">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加备忘录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_all_work">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查看备忘录</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_work_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>81</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="today_date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>今天是：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>查看所有备忘录：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>日程安排：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
